### ISSUE TITLE
Dataset labels

### ISSUE THREAD

**Issue by 1024-m** [RegNLP/ObliQADataset#1]
Hi, The training dataset has just the {questionID, question, passages{documentID, passageID, passage}, and group}, where can we access the answers i.e labels of training data for task 2 ?

**Comment by yash98** [comment]
This uses a reference-free metric for evaluating answers which means it does not need ground truth answers. It uses just the ground truth passages to evaluate the generated answer

checkout repass metric in the paper for more details: https://arxiv.org/abs/2409.05677
